coreos-base/oem-packet/files/reset-interfaces.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+logger "Hard reset of all network interfaces due to upstream systemd-networkd issue: https://github.com/coreos/bugs/issues/36 ."
+ip link set enp1s0f0 down
+ip link set enp1s0f1 down
+ip link del bond0
+systemctl restart systemd-networkd
+sleep 15
```

Comment:
The virtual interface takes time to come up and systemctl seems to return after kicking it off. This makes sense, because systemctl has finished it's job, it is ambivalent success/failure of the the service it calls, and the subsequent job that announces the machine to our API requires network.  

I can think of two alternatives.  I can force the second job to wait until network is up, and is probably better than this sleep.  

Or I can make a target that is statisfied by the bond0 having link.